src/mol-plugin-ui/spec.ts

```diff
@@ -24,6 +25,17 @@ interface PluginUISpec extends PluginSpec {
             view?: React.ComponentClass | React.FC,
             controls?: React.ComponentClass | React.FC,
             snapshotDescription?: React.ComponentClass | React.FC,
+            removeGranularity?: boolean,
+            removeUnion?: boolean,
+            removeSubtract?: boolean,
+            removeIntersect?: boolean,
+            removeSet?: boolean,
+            removeBrush?: boolean,
+            removeCubeOutline?: boolean,
+            removeRemove?: boolean,
+            removeRestore?: boolean,
+            removeHelpOutline?: boolean,
+            removeCancelOutlined?: boolean,
```

Comment:
Can we make this something like this instread?

```ts
components?: {
   selectionTools?: {
      control?: React.ComponentClass | React.FC,
      hide?: {
         granularity?: boolean,
         // ....
      }
   }
}
```

Reply:
Sure! I have updated with your suggestion. Please let me know if I have executed it correctly.